README.md

```diff
@@ -206,6 +206,55 @@ place = client.place_update 'home', 'my address'
 place.address #=> retuns fully qualified address of location
 ```
 
+### Retrieve a reminder
+This allows you to get the status of an existing ride reminder.    
+**Note**: It only supports _server_token_. 
+```ruby
+client = Uber::Client.new do |config|
+  config.client_id     = "YOUR_CLIENT_ID"
+  config.client_secret = "YOUR_CLIENT_SECRET"
+  client.server_token  = "YOUR_SERVER_TOKEN"
+end
+client.reminder 'reminder_id'
+#=> Uber::Reminder
+```
+
+### Add a reminder
+This allows developers to set a reminder for a future trip. You can pass object of `Time` or _unixtime_ in `reminder_time` and `event.time`.
+```ruby
+reminder = client.add_reminder({reminder_time: Time.local(2016, 9, 8, 23, 23, 23),
+                                phone_number: '+91-9999999999',
+                                trip_branding: {link_text: 'My first reminder'},
+                                event: {time: Time.now + 234234},
+                                reminder_id: 'rem1' })
+reminder.event.time
+#=> 2016-09-11 11:02:06 UTC
+reminder.reminder_time
+#=> 2016-09-08 17:53:23 UTC
+reminder.reminder_stauts
```

Comment:
oops. fixed.

On Thu, Sep 8, 2016 at 11:39 PM, Charlyn G <notifications@github.com> wrote:

> In README.md
> <https://github.com/sishen/uber-ruby/pull/23#discussion_r78059333>:
>
> > +#=> Uber::Reminder
> > +```
> > +
> > +### Add a reminder
> > +This allows developers to set a reminder for a future trip. You can pass object of `Time` or _unixtime_ in `reminder_time` and `event.time`.
> > +```ruby
> > +reminder = client.add_reminder({reminder_time: Time.local(2016, 9, 8, 23, 23, 23),
> > +                                phone_number: '+91-9999999999',
> > +                                trip_branding: {link_text: 'My first reminder'},
> > +                                event: {time: Time.now + 234234},
> > +                                reminder_id: 'rem1' })
> > +reminder.event.time
> > +#=> 2016-09-11 11:02:06 UTC
> > +reminder.reminder_time
> > +#=> 2016-09-08 17:53:23 UTC
> > +reminder.reminder_stauts
>
> reminder_stauts -> reminder_status
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sishen/uber-ruby/pull/23/files/ff98ef133700841d1f3228a2ac7ef82923ccfe13#r78059333>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgZot5KQQF_XypQCzxxJi5TifFyXwEGks5qoE93gaJpZM4J4M1c>
> .
>



-- 
-Ankur Goel